--- jpilot-syncmal-0.80/configure.in	2007-01-16 15:12:27.000000000 -0800
+++ jpilot-syncmal-0.80/configure.in.new	2007-07-25 13:23:38.000000000 -0700
@@ -58,7 +58,7 @@ libmallibs=${FORCE_LIBMAL_LIBS:-no}
 LIBMAL_LIBS="-lmal"
 
 if test $libmallibs = no; then
-    for libmal_libs in $libmal_prefix/lib /usr/lib /usr/local/lib; do
+    for libmal_libs in $libmal_prefix/lib /usr/lib /usr/local/lib $libmal_prefix/lib64 /usr/lib64 /usr/local/lib64; do
         if test -r "$libmal_libs/libmal.so" ; then
             libmallibs=yes
             LIBMAL_LIBS="-L$libmal_libs $LIBMAL_LIBS"
@@ -122,7 +122,7 @@ pilotlibs=${FORCE_PILOT_LIBS:-no}
 PILOT_LIBS="-lpisock"
 
 if test $pilotlibs = no ; then
-    for pilot_libs in $pilot_prefix/lib /usr/lib /usr/local/lib/ \
+    for pilot_libs in $pilot_prefix/lib /usr/lib /usr/local/lib $pilot_prefix/lib64 /usr/lib64 /usr/local/lib64 \
         /usr/extra/pilot/lib ; do
         if test -r $pilot_libs/libpisock.so >/dev/null 2>&1 ; then
             pilotlibs=yes
